src/tests/multihost/alltests/test_ns_account_lock.py

```diff
@@ -10,8 +10,9 @@
 import pytest
 import time
 import paramiko
-from sssd.testlib.common.utils import LdapOperations
-from sssd.testlib.common.utils import SSHClient
+from sssd.testlib.common.utils import LdapOperations, \
+    sssdTools, \
+    SSHClient
```

Comment:
Line 14 and 15 can be on the same line. Line length has been increased to 119 characters

Reply:
changed

Reply:
Sorry, I meant same line as `LdapOperations`

Reply:
changed